experimental/SetPartitions/src/PartitionProperties.jl

```diff
@@ -0,0 +1,193 @@
+function size(p::SetPartition)
+    length(p.lower_points) + length(p.upper_points)
+end
+
+function is_composable(p::SetPartition, q::SetPartition)
+    length(p.upper_points) == length(q.lower_points)
```

Comment:
```suggestion
    return length(p.upper_points) == length(q.lower_points)
```
All functions with the keyword `function` in front of should use `return`.